css/index.css

```
@@ -1,4 +1,200 @@
+/*mobile Style*/
+#nav {
+list-style: none;
+background-color: #f5f5dc;
+border-radius: 6px;
+box-shadow: 0 4px 8px rgb(231, 33, 225);
+}
+#nav a {
+text-decoration: none;
+
+}
 body {
+    background-color: rgb(98, 174, 122);
+    color: blue;
+}
+
+section {
+    margin-bottom: 50px;
+}
+
+#About {
+    text-align: center;
+}
+
+h1 {
+    color: navy;
+    font-size: 30px; 
+    font-weight: bold;
+    font-family: Cedarville Cursive;
+}
+
+h2 {
+    color: navy;
+    font-size: 24px; 
+    font-weight: bold;
+}
+
+#Experience {
+    display: flex;
+    flex-direction: column;
+}
+
+#Experience ul {
+    display: flex;
+    flex-direction: column;
+    gap: 25px;
+}
+
+.experience-item {
+    display: flex;
+    flex-direction: column;
+    border-radius: 6px;
+    background-color: #f5f5dc;
+    box-shadow: 0 4px 8px rgb(231, 33, 225);
+}
+
+.job-header {
+    display: flex;
+    justify-content: space-between;
+}
+
+.job-title {
+    font-weight: bold;
+}
+
+.job-dates {
+    font-style: italic;
+}
+
+#Connect a {
+    background-color: antiquewhite;
+    box-shadow: 0 2px 4px rgba(231, 33, 225);
+}
+
+.row {
+    padding: 10px;
+}
+
+#Connect i {
+    margin: 3px;
+    font-size: 24px; 
+}
+
+#Connect ul {
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+}
+
+
+/*Tablet style*/
+@media only screen and (min-width: 600px) and (min-width: 900px) {
```

You're using two different values for `min-width`, that condition will never be fulfilled. I think you meant to use `max-width` for the second one.

```suggestion
@media only screen and (min-width: 600px) and (max-width: 900px) {
```

---

thank you! I will get right on this!